plugins/cms-export/src/csv.ts

```diff
@@ -54,9 +54,15 @@ function isFieldSupported(field: Field): field is SupportedField {
     }
 }
 
-export function getDataForCSV(slugFieldName: string | null, fields: Field[], items: CollectionItem[]): Rows {
+export function getDataForCSV(
+    slugFieldName: string | null,
+    fields: Field[],
+    items: CollectionItem[],
+    limit?: number
+): Rows {
     const rows: Rows = []
     const supportedFields = fields.filter(isFieldSupported)
+    const limitedItems = typeof limit === "number" ? items.slice(0, limit) : items
```

Comment:
Nit: This is fine, but could also do an early return in the `for-of` below, where it compares `rows.length - 1` (to account for the header) to `limit`.